apteryx-xml.h

```diff
@@ -181,6 +184,7 @@ GList *sch_parm_removes (sch_xml_to_gnode_parms parms);
 GList *sch_parm_creates (sch_xml_to_gnode_parms parms);
 GList *sch_parm_replaces (sch_xml_to_gnode_parms parms);
 void sch_parm_free (sch_xml_to_gnode_parms parms);
+sch_node *sch_node_ns_child (xmlNs *ns, sch_node * parent, const char *child);
```

Comment:
sch_node *sch_node_child_ns (sch_node * parent, const char *child, const char *namespace);
and place below 
sch_node *sch_node_child (sch_node *parent, const char *name);

Reply:
The function sch_node_child_ns has the arguments
sch_node *sch_node_child_ns (xmlNs *ns, sch_node * parent, const char *child);
Because it is passing a namespace (xmlNs *ns) it needs to be in the section ifdef APTERYX_XML_LIBXML2 or is my understanding not correct.

Reply:
hence my suggestion to change the arguments ... const char *namespace

Reply:
I have a valid xmlNs in netconf from the result XML data. If I only pass a namespace string to sch_node_child_ns then I can not create another xmlNs without a valid xmlNode to attach it to.